viz.yaml

```diff
@@ -41,5 +44,12 @@ process:
     depends: 
       waterUse: waterUse-data
 visualize:
+  -
+    id: wateruse-treemap
+    location: cache/treemaps/
+    visualizer: treemap
+    scripts: scripts/visualize/treemap.R
+    depends:
+      waterUse_clean: calc-waterData
```

Comment:
this is a general vizlab question: what's the purpose of renaming viz items, if that's what is happening here?